topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+---
+layout: tutorial_hands_on
+
+title: Pangeo Notebook in Galaxy - Introduction to Xarray
+zenodo_link: 'https://doi.org/10.5281/zenodo.5805953'
+requirements:
+- 
+  type: internal
+  topic_name: data-science
+  tutorials:
+    - python-basics
+- 
+  type: internal
+  topic_name: data-science
+  tutorials:
+    - python-plotting
+    - python-advanced-np-pd
+- 
+  type: "external"
+  title: Data Carpentry Geospatial Workshop
+  link: https://datacarpentry.org/geospatial-workshop/
+follow_up_training:
+- 
+  type: "external"
+  title: Pangeo Tutorial Gallery
+  link: http://gallery.pangeo.io/repos/pangeo-data/pangeo-tutorial-gallery/index.html
+- 
+  type: "external"
+  title: Pangeo Gallery
+  link: http://gallery.pangeo.io/index.html
+questions:
+- What is Pangeo notebook?
+- How to start Pangeo Notebook in Galaxy?
+- What are the main software components of the Pangeo ecosystem?
+- What is Xarray?
+- How to manipulate Xarrays?
+- How to print metadata information?
+- How to make a selection?
+- How to visualize?
+- How to filter? 
+- How to make reduction operations (mean, max, min)?
+- How to resample my data?
+- Where to go next?
+objectives:
+- Learn to get metadata information using Xarray Galaxy Tools
+- Learn to select data
+- Learn to visualize data
+- Learn to filter, make reduction operations (mean, max, min)
+- Learn to resample my data
+- Learn to cite and contribute to Pangeo
+time_estimation: 1H
+key_points:
+- Pangeo and Xarray
+- Read, select, mask and plot netCDF data using Xarray
```

Comment:
Perhaps the key points could be explained a bit more.

Reply:
You are right. I will revise the key points and make them more explicit.